notebooks/2_PrincipalCovariatesRegression.ipynb

```diff
@@ -83,23 +83,32 @@
     "(1-\\alpha){\\left\\lVert \\mathbf{Y} - \\mathbf{X}\\mathbf{P}_{XT}\\mathbf{P}_{TY}\\right\\rVert^2}.\n",
     "\\end{equation}\n",
     "\n",
-    "It is easier to minimize this by looking for a projection $\\tilde{\\mathbf{T}}$ in a latent space for which we enforce orthonormality, $\\tilde{\\mathbf{T}}^T\\tilde{\\mathbf{T}} = \\mathbf{I}$\n",
+    "When $\\alpha = 1$, $\\ell = {\\left\\lVert \\mathbf{X} - \\mathbf{X}\\mathbf{P}_{XT}\\mathbf{P}_{TX}\\right\\rVert^2}$. It is equivalent to  principal component analysis (PCA). When $\\alpha = 0$, $\\ell = {\\left\\lVert \\mathbf{Y} \\mathbf{X}\\mathbf{P}_{XT}\\mathbf{P}_{TY}\\right\\rVert^2}$. It is equivalent to ridge regression.\n",
+    "\n",
+    "It is easier to minimize this by looking for a projection $\\tilde{\\mathbf{T}}$ in a latent space if we enforce the column vectors in $\\tilde{\\mathbf{T}}$ to be orthonormal, i.e., $\\tilde{\\mathbf{T}}^T\\tilde{\\mathbf{T}} = \\mathbf{I}$. Note that $\\tilde{\\mathbf{T}}\\tilde{\\mathbf{T}}^T \\neq \\mathbf{I}$ since the size of  $\\tilde{\\mathbf{T}}$ is $n_{samples}\\times n_{PCA} $and $n_{samples} > n_{PCA}$.\n",
```

Comment:
This comment is not necessary for the understanding of the reader and may confuse more than it helps.

Reply:
I added this based on my experience. For some readers like me, this will help them figure it out.

Reply:
@bhelfrecht @Luthaf thoughts?

Reply:
Okay, I will remove it.